packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -1184,46 +845,48 @@ contract('FederatedAttestations', (accounts: string[]) => {
     })
   })
 
-  describe('#deleteAttestation', () => {
+  describe('#revokeAttestation', () => {
     beforeEach(async () => {
       await accountsInstance.authorizeSigner(signer1, signerRole, { from: issuer1 })
       await accountsInstance.completeSignerAuthorization(issuer1, signerRole, { from: signer1 })
       await federatedAttestations.registerAttestation(
         identifier1,
         issuer1,
         account1,
-        nowUnixTime,
         signer1,
+        nowUnixTime,
```

Comment:
Nit consider affirming `attestationInStorage` and other state changes in the beforeEach hook or if we're worried about efficiency in a 'check state is setup correctly' test that just runs once

Reply:
I think this is covered by the last test `should modify identifierToAddresses and addresstoIdentifiers accordingly`